chapter07/dcox/ex7_4

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+while (<STDIN>) {
+    if (/[A-Z]+[a-z]+[a-zA-Z]*/) { print $_; }
```

Comment:
Why must the word have a capital before the lowercase letters, and what about non-letter symbols?